.github/workflows/test.yml

```diff
@@ -150,7 +150,7 @@ jobs:
           python -m pip install tox
 
       - name: Run tox docs target
-        run: make man
+        run: tox -e docs
```

Comment:
why not use a make target for this?

Reply:
Why? It's an unnecessary level of indirection. The other test files run tox directly, too.

Reply:
It is an unnecessary level of indirection but it helps us validate the Make file at the same time.